llpc/test/shaderdb/OOB_Check_Load_Nested_Struct_lit.frag

```diff
@@ -0,0 +1,38 @@
+#version 450 core
+
+layout (location = 0) in vec4 inColor;
+layout (location = 0) out vec4 outFragColor;
+layout (binding = 0) uniform Constant { int array_index; } c;
+
+struct NestedStruct {
+  vec4 nested[4];
+};
+
+struct BaseStruct {
+	vec4 data[10];
+	NestedStruct data2[12];
+};
+
+void main() {
+  BaseStruct checker[5];
+
+  outFragColor = inColor * checker[c.array_index].data[c.array_index + 1] + checker[c.array_index].data2[c.array_index + 2].nested[c.array_index];
+}
+
+// BEGIN_SHADERTEST
+/*
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s -scratch-bounds-checks | FileCheck -check-prefix=SHADERTEST %s
```

Comment:
Same comments as in the other file. In addition, could you add a brief comment at the top of the file saying what this is exactly testing?